.github/workflows/build-docs.yml

```
@@ -0,0 +1,15 @@
+name: Build docs
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  build-docs:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ammaraskar/sphinx-action@master
```

We should use a fixed version, current lastest is 0.4
```suggestion
      - uses: ammaraskar/sphinx-action@0.4
```